src/tailwind.css

```diff
@@ -12,6 +12,15 @@
   h3 {
     font-size: 28px;
   }
+
+  p {
+    @apply mt-0;
+    @apply mb-4;
+  }
```

Comment:
u sure u wanna put it here to make it global?

Reply:
um yeah im not sure about that too, ok let me think more on this

Reply:
k reverted